src/Uno.UI.FluentTheme.v1/Resources/Version1/PriorityDefault/CommandBar_themeresources.xaml

```diff
@@ -955,7 +1050,7 @@
         </Setter>
     </Style>
 
-    <Style x:Key="CommandBarWithoutRevealStyle" TargetType="CommandBar">
+    <Style x:Key="CommandBarWithoutRevealStyle" TargetType="muxc:CommandBar">
```

Comment:
I probably forgot the reason for this but - why does `CommandBar` need to be in MUX namespace? The problem I see is that this way it will not work in Uno.UI, only in Uno.WinUI. If I am not wrong, it would mean that this control would never work in UWP (because trying to use `muxc:CommandBar` in XAML would fail during compilation). Wouldn't it be possible to completely replace the fake `CommandBar` we have now with this one and then possibly move the existing `CommandBar` implementation to something like `UnoCommandBar`? I understand this would be a bigger breaking change, but we will have 4.0, so it would probably be worth it. The ported code is not technically a MUX `CommandBar` (akin to `TreeView` or `TabView`), but rather the WUX `CommandBar` - the control did not change in MUX and is not specific to it. @jeromelaban

Reply:
The only way this could then be used would be something like:

```
<not_win:Grid>
    <muxc:CommandBar x:Name="MyCommandBar" ... />
</not_win:Grid>
<win:Grid>
    <CommandBar x:Name="MyCommandBar" ... />
</win:Grid>
```

Which would work, but it would be quite ugly, especially as `CommandBar` is one of the controls which are used quite often...

Reply:
Maybe both the Uno-legacy `CommandBar` and WinUI/UWP CommandBar could live in one class too (similarly as we have the "native" `Button`, "native` `ToggleSwitch`, etc. supported)

Reply:
Ah, I notice now that @davidjohnoliver had similar questions 😀

Reply:
Yeah I agree it's going to be ugly to use with Windows :( But we didn't want to make a breaking change by moving the current WUX CommandBar since it is used for native UINavigationBar/Toolbar rendering on iOS/Android.

@jeromelaban may be able to weigh in here. But I am still open to having the current CommandBar be moved to a Legacy namespace so the one true WinUI CommandBar can live in WUX